app/services/auth.service.ts

```
@@ -6,181 +1,366 @@
+  country: string;
+  avatar?: File;
 }
 
 interface LoginData {
-    email: string;
-    password: string;
+  email: string;
+  password: string;
 }
 
 interface AuthResponse {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'AuthResponse'. Consider to delete it or rename it.

'AuthResponse' is defined here:


(lint/suspicious/noRedeclare)